docs/related/amf-conversion.mdx

```diff
@@ -1,9 +1,193 @@
+This is the reason why in a *cross parsing-rendering process*, the AMF graph needs to go through a transformation called *“Compatibility resolution”*.
+
+**Compatibility resolution** is a specific resolution pipeline that is used to adapt an API graph parsed from spec A, to fulfill the requirements of the spec in which it will be rendered.
+An example of how to use Compatibility Resolution can be seen in [Example 1](#conversion-examples).
+
+
+How does this impact the API model? For example, fields that are not required in the origin spec but required in the target spec,
+will be added with default values if they are not present in the origin spec.
+
+:::tip The Objective
+The objective of conversion is to **output a valid AMF model for the spec in which it will be rendered**.
+It aims to **keep information loss to a minimum through a best-effort process** with some limitations.
+:::
+
+
+## Conversion Goals
+
+Conversion resolution has several goals:
+- Make transitioning to/from OAS to RAML easier for the end user.
+- Give the user an approximate view of what his API in another spec would look like
+- Let tools use specs that they don’t directly support
+- Take advantage of spec-specific tooling
+    - for example, converting to OAS to use swagger-ui in a microservice or using a trusted client-spec generator that only works with RAML.
+
+
+
+# Conversion limitations
+The AMF team, having listened to other team’s needs, is the one that dictates how the model is adapted to fit other specs:
+- If there is 1:1 information type mapping between specs it will be converted.
+- If there are missing mandatory fields default value will be added. The default’s value is opinionated.
+- Anything else is omitted.
+
+
+### Concepts present in a spec but nonexistent in another aren’t supported
+Sometimes a spec may introduce concepts that don’t exist in other specs. There are many cases coming from Raml to Oas, some are:
+- Datatype Fragments
+- Libraries
+- Resource Types and Traits
+- Overlays and Extensions
+
+These document types and objects cannot be converted to Oas as these constructs do not exist in that spec.
+These constructs can be migrated to Oas if and only if they are referenced from a root Api document.
+
+
+### Not all specs are supported
+
+This is a direct consequence of the previous limitation, not all specs can be converted between each other. Currently, Async API is not supported for conversion.
+It’s event-led concepts don’t have a direct correspondence to REST concepts described in RAML or OAS.
+For example, how would an Async APIs “subscribe” or “binding” be in RAML? The amount of similar concepts is simply not enough to create similar graphs.
+Supported conversions are:
+- RAML 1.0 to OAS 2.0 and vice-versa
+- RAML 1.0 to OAS 3.0 and vice-versa
+
+
+### Information may be lost
+Original information that is incompatible with the target spec may be lost. This is also a consequence of the first limitation.
```

Comment:
Consider briefly re-describing the first limitation, so that the reader doesn't have to scroll back up.